tests/run-make/core-no-fp-fmt-parse/rmake.rs

```diff
@@ -0,0 +1,21 @@
+// ignore-tidy-linelength
```

Comment:
Why the ignore? Is this just for the comment on (current) line 3? Can we wrap that comment at ~100 characters instead?

Reply:
Indeed, that is what it was for. I have removed the ignore and wrapped the comment.